presto-prometheus/src/test/java/io/prestosql/plugin/prometheus/TestPrometheusQueryVectorResponseParse.java

```diff
@@ -27,6 +27,7 @@
 import static org.testng.Assert.assertNotNull;
 import static org.testng.Assert.assertTrue;
 
+@Test(singleThreaded = true) // see @BeforeMethod
```

Comment:
maybe we don't need `@BeforeMethod` here, but only `@BeforeClass`

Reply:
chose to ignore for now